agent/connect/ca/provider_vault.go

```diff
@@ -94,7 +115,7 @@ func (v *VaultProvider) Configure(cfg ProviderConfig) error {
 	if err != nil {
 		return err
 	} else if secret == nil {
-		return fmt.Errorf("Could not look up Vault provider token: not found")
+		return fmt.Errorf("could not look up Vault provider token: not found")
```

Comment:
here and on line 142:

Generally, I try not to modify err strings on the offchance that any callees actually do string processing. 

Here I think it's pretty safe to do it (as opposed to out of some API endpoint) but just wanted to call it out.